internal/router/middlewares/logging.go

```diff
@@ -9,17 +10,20 @@ import (
 // WithLogging logs requests and responses that contain useful information.
 func WithLogging(h http.Handler) http.Handler {
 	handler := func(rw http.ResponseWriter, r *http.Request) {
+		clientIP, err := extractClientIP(r)
+		if err != nil {
+			log.Warn().Err(err).Msg("can't extract client ip")
+			clientIP = "none"
```

Comment:
Uhm, should we reconsider this existing `"none"` value to be aligned with [this](https://github.com/tablelandnetwork/go-tableland/pull/385/files#diff-5588b773fe67092e7366bf6608cb74cae8d1475902b773f5c18a87e305b50dc2R276)? Like, setting it `""` (empty)?

No strong opinions, but want to be sure your `!=""` intention wasn't `!="none"` (or switch `"none"` to `""`.)

Reply:
Nice catch! Thanks